ODIN_II/SRC/memories.cpp

```diff
@@ -437,6 +437,8 @@ void split_sp_memory_depth(nnode_t *node, int split_size)
 
 		npin_t *pin = signals->out->pins[i];
 		pin->name = mux->name;
```

Comment:
Should check that the name is null before assignment otherwise it should be freed beforehand